### PR TITLE
Add cookie prompt management experiment

### PR DESCRIPTION
**Pull request by joshliebe**
Task/Issue URL: https://app.asana.com/0/488551667048375/1203956750670569/f

### Description
This PR adds a cookie prompt management experiment.

### Steps to test this PR

_Change all variants to 0 except for the `mr` variant_

_Non-European Country_
- [x] Set your device region to US/en
- [x] Install from this branch
- [x] The app should have the default variant
- [x] Visit cnn.com and bitcoin.org
- [x] Verify no cookie prompt behavior

_European Country_
- [x] Set your region to ES/en
- [x] Install from this branch
- [x] The app should have the `mr` variant
- [x] Visit cnn.com and bitcoin.org
- [x] Verify that the cookie prompt management works as expected

### CODE REVIEW

File: statistics/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt [hunk 1]
```diff
@@ -73,9 +73,57 @@ interface VariantManager {
             "GB",
         )
 
-        private val usAndCaCountries = listOf(
-            "US",
-            "CA",
+        private val EuropeanCountries = listOf(
```
Comment (joshliebe) [inline statistics/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt:76]: List taken from: https://github.com/duckduckgo/iOS/blob/f5a0879d192e59e25104bfbe5a085c208ab2a187/Core/LocaleExtension.swift#L24

Comment (joshliebe) [inline statistics/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt:76]: List taken from: https://github.com/duckduckgo/iOS/blob/f5a0879d192e59e25104bfbe5a085c208ab2a187/Core/LocaleExtension.swift#L24

File: statistics/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt [hunk 2]
```diff
@@ -64,6 +73,59 @@ interface VariantManager {
             "GB",
         )
 
+        private val europeanCountries = listOf(
```
Comment (joshliebe) [inline statistics/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt:76]: List taken from: https://github.com/duckduckgo/iOS/blob/f5a0879d192e59e25104bfbe5a085c208ab2a187/Core/LocaleExtension.swift#L24

**Review by marcosholgado — APPROVED** [review]
LGTM! Nice job!

**Pull request merged**